// [[Reason: TODO Need to fix DTE looking recursively for assigned members.]] ////

//// [tests/cases/conformance/salsa/typeFromPropertyAssignment36.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -13,14 +13,13 @@
 typeFromPropertyAssignment36.ts(32,12): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 typeFromPropertyAssignment36.ts(34,10): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 typeFromPropertyAssignment36.ts(34,10): error TS9009: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
 typeFromPropertyAssignment36.ts(42,3): error TS2565: Property 'q' is used before being assigned.
-typeFromPropertyAssignment36.ts(59,7): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
-typeFromPropertyAssignment36.ts(59,7): error TS9009: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
+typeFromPropertyAssignment36.ts(59,11): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 typeFromPropertyAssignment36.ts(64,3): error TS2565: Property 'expando' is used before being assigned.
 
 
-==== typeFromPropertyAssignment36.ts (9 errors) ====
+==== typeFromPropertyAssignment36.ts (8 errors) ====
     function f(b: boolean) {
              ~
 !!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
         function d() {
@@ -90,12 +89,10 @@
     d.r
     
     // test function expressions too
     const g = function() {
-          ~
+              ~~~~~~~~
 !!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
-          ~
-!!! error TS9009: Assigning properties to functions without declaring them is not supported with --isolatedDeclarations. Add an explicit declaration for the properties assigned to this function.
     }
     if (!!false) {
         g.expando = 1
     }
